about.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Lato&display=swap" rel="stylesheet">
+
+    <!-- Font Awesome -->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css"
+        integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+    <title>Document</title>
+    <link rel="stylesheet" href="./css/aboutpage.css">
+    <link rel="stylesheet" href="/css/aboutheader.css">
+    <link rel="stylesheet" href="/css/footer.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+</head>
+
+<body>
+    <header>
+        <div class="topnav">
+            <nav>
+                <li><i class="fa-brands fa-instagram-square"></i></li>
+                <li> <i class="fa-brands fa-twitter"></i></li>
+                <li><i class="fa-brands fa-facebook"></i></li>
+            </nav>
+        </div>
+        <div class="mainnav">
+            <h1 class="main-h3"> The chance</h1>
```

Comment:
- [ ] Please make the logo be a link to the home page.
- [ ] Please show the whole logo as there is part of the logo that is not showen.
![Screenshot from 2022-05-19 23-39-22](https://user-images.githubusercontent.com/33752735/169409181-9a76602d-5172-447a-91ae-cb79a1fb503f.png)